index.js

```diff
@@ -0,0 +1,274 @@
+//задачи на условное ветвление
+//task1 Если переменная a равна 10, то выведите 'Верно', иначе выведите 'Неверно'.  
+let a = 10;
+if (a == 10) {
+  alert ("True");
+} else {
+  alert ("False");
+}
+
+//task2 В переменной min лежит число от 0 до 59. Определите в какую четверть часа попадает это число (в первую, вторую, третью или четвертую).
+let min = 10;
+if (min >= 0 && min <= 14) {
+  alert ("В первую четверть.");
+}
+
+if (min >= 15 && <= 30) {
+  alert ("Во вторую четверть.");
+}
+
+if (min >= 31 && <= 45) {
+  alert ("В третью четверть.");
+}
+
+if (min >= 46 && <= 59) {
+  alert ("В четвертую четверть.");
+}
+
+//task3 Если переменная a равна нулю, то выведите 'Верно', иначе выведите 'Неверно'. Проверьте работу скрипта при a, равном 1, 0, -3.
+let a = 0;
+if (a == 10) {
+  alert ("True");
+} else {
+  alert ("False");
+}
+
+let a = 1;
+if (a == 1) {
+  alert ("True");
+} else {
+  alert ("False");
+}
+
+let a = 0;
+if (!a) {
+  alert ("True");
+} else {
+  alert ("False");
+}
+
+let a = -3;
+if (a == -3) { 
+  alert ("True");
+} else {
+  alert ("False");
+}
+
+//task4 Если переменная a больше нуля, то выведите 'Верно', иначе выведите 'Неверно'. Проверьте работу скрипта при a, равном 1, 0, -3.
+let a > 0;
+if (a > 0) {
+  alert ("True");
+} else {
+  alert ("False");
+}
+
+let a = 1;
+if (a == 1) {
+  alert ("True");
+} else {
+  alert ("False");
+}
+
+let a = 0;
+if (!a) {
+  alert ("True");
+} else {
+  alert ("False");
+}
+
+let a = -3;
+if (a == -3) { 
+  alert ("True");
+} else {
+  alert ("False");
+}
+
+
+//задачи на логические операторы
+//task 1 Если переменная a больше нуля и меньше 5-ти, то выведите 'Верно', иначе выведите 'Неверно'. Проверьте работу скрипта при a, равном 5, 0, -3, 
+let a > 0 && a < 5;
+if (a > 0 && a < 5) {
+  alert ("True");
+} else {
+  ("False");
+}
+
+let a = 5;
+if (a > 0 && a < 5) {
+  alert ("True");
+} else {
+  alert ("False");
+}
+
+let a = 0;
+if (a > 0 && a < 5) {
+  alert ("True");
+} else {
+  alert ("False");
+}
+
+let a = -3;
+if (a > 0 && a < 5) {
+  alert ("True");
+} else {
+  alert ("False");
+}
```

Comment:
you don't need to copy paste code, you can just check it by changing value of "a"

and you can't declare more than ONE variable with the same name!

so the code is wrong